### PR TITLE
Add css linting rules to run with `npm test` and `npm run dev`

### PR DESCRIPTION
**Pull request by BrendanFDMoore**
Goals from @SethDavenport were to apply css linting rules with `stylelint` that would:

1. Cause `npm test` to fail with errors to block CI builds
2. Show rule violation warnings at console while using `npm run dev`

This change set achieves both, I believe in a minimal way. 

If accepted, this will also be added to all other starters.

Connected to rangle/rangle-starter#51

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,9 @@
     "cover": "remap-istanbul -i coverage/chrome/coverage-final.json -o coverage/coverage.json",
     "postcover": "istanbul report lcov text-summary",
     "dev": "cross-env NODE_ENV=development webpack-dev-server -d --inline --progress --no-info",
-    "lint": "tslint './src/**/*.ts'",
+    "lint": "npm run lint-ts && npm run lint-css",
```
Comment (JaKXz) [inline package.json]: Can we use a package like [concurrently](https://www.npmjs.com/package/concurrently) for running lint tasks (and others that can happen in parallel) via npm scripts?

Comment (BrendanFDMoore) [inline package.json]: @JaKXz Yes, if that would be a preferred structure. I spoke with @SethDavenport about normalizing the structures using in `webpack.config.js` (eg, external `loaders.js` file), and we think that it makes sense to have them all as similar as possible. I think doing some tidying in `package.json` like you suggest makes sense as part of that effort.

Comment (SethDavenport) [inline package.json]: @JaKXz I don't believe `concurrently` adds value in this case.  The linting tasks are relatively quick, so running them sequentially is not a bottleneck.

Furthermore, running them concurrently will interleave the output from the two commands in a confusing way.

**Comment by SethDavenport** [comment]
:shipit:

**Comment by SethDavenport** [comment]
@BrendanFDMoore: note the 'connected to' comment I added - this links the PR with the associated waffle board issue, which makes things a bit easier to track.

**Pull request merged**